### PR TITLE
Add usage sample to README

### PR DESCRIPTION
**Pull request by tstenner**
Add a short usage example to the README, showing how to load an xdf file and plot its contents.

### CODE REVIEW

**Comment by cbrnr** [comment]
Looks good, but can you use 4 spaces to indent blocks?

**Comment by tstenner** [comment]
Done, I also removed the text from the marker display because it'd add one legend entry per marker, not unique marker.

File: README.md [hunk 1]
```diff
@@ -8,6 +8,32 @@ pyXDF
...
+data, header = pyxdf.load_xdf('test.xdf')
+
+for stream in data:
+    y = stream['time_series']
+
+    if isinstance(y, list):
+        # list of strings, draw one vertical line for each marker
+        for timestamp, marker in zip(stream['time_stamps'], y):
+            plt.axvline(x=timestamp)
+            print(f'Marker "{marker}" @ {timestamp}s')
```
Comment (cbrnr) [inline README.md]: Maybe use `print(f'Marker {marker[0]} @ {timestamp:.2f}s')` for a nicer display?

Comment (cbrnr) [reply README.md]: Or if you want the quotes around the marker name:

```python
print(f'Marker "{marker[0]}" @ {timestamp:.2f}s')
```

**Comment by cbrnr** [comment]
Ready to merge @tstenner?

**Pull request merged**